flytekit-java/src/main/java/org/flyte/flytekit/SdkWorkflow.java

```diff
@@ -63,14 +77,15 @@ public SdkNode<OutputT> apply(
             .build();
 
     Map<String, SdkBindingData<?>> outputs =
-        innerBuilder.getOutputs().entrySet().stream()
+        getSdkBindingDataOutputs().entrySet().stream()
```

Comment:
```suggestion
        sdkBindingDataOutputs.entrySet().stream()
```
We can use the field directly